### DISCUSSION TITLE
Docs: Easy flow for macOS use

### DISCUSSION THREAD

**Discussion by jasikpark** [davidlday/vscode-languagetool-linter#771] [LanguageTool Installation]
I think this is mostly covered by your docs, but not entirely clearly, since I originally thought I needed to manage java.

I got setup w/ this tool by:

1. Installing the extension from the VSCode extension marketplace
2. "Using" the linter for a few days while it was unattached to a service
3. Actually reading the README and realizing I needed to install the languagetools service
4. Running `brew install languagetools` which installed the bottle version of languagetool, rather than the app
5. Running `brew services restart languagetools` which started up the languagetools service that got installed by homebrew with the tool
6. CTRL+S on the MDX file I had open in VSCode, and I'm off to the races!!

Thanks for working on this tool! So much better than `cspell` and trying to manage an ever-growing dictionary of words cspell doesn't know.

**Comment by davidlday** [comment]
Thanks so much for the feedback! The docs definitely need some work. I'll get this incorporated as soon as I'm able. Might even pop open a "getting started" screen to give folks platform-specific instructions as install time. Assigned to myself, and leaving it open to keep this on my radar.

**Comment by jasikpark** [comment]
This extension has been really great! It stumbles over quite a bit of MDX syntax, but it's nice to have the grammar and spelling checks when it's available -- "add to workspace ignores" is much more enjoyable than manually editing a cspell config too :p
